openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -142,7 +139,7 @@ def _create_placeholder_name(self, placeholder_data):
         # add loader arguments if any
         loader_args = placeholder_data["loader_args"]
         if loader_args:
-            loader_args = json.loads(loader_args.replace('\'', '\"'))
+            loader_args = json.loads(loader_args.replace("'", '"'))
```

Comment:
This PR seems to implement tons of cosmetic style changes (some if which the Hound disagrees with too). Anyway, the cosmetic changes make it hard to see what actually changed for this PR. A quick read on my end makes me assume that **only** this line really was a change to fix an issue. Is that correct?

At this point in time I'm not sure why we need to even do the replace thing to begin with. Why is that?

Reply:
Yes it does, I make bad manipulation while using a formatter to correct a specific line. Let me correct that

Reply:
I'd say a direct hard replacement of quotes isn't safe to begin with - but I can see how an artist might think using single quotes is fine even though they should've been writing double quotes for JSON.

However, a JSON string like `"my neighbour's cat" would now break due to this string replacement - breaking esstentially what is valid JSON to begin with.